Solgema/fullcalendar/browser/icalexport.py

```diff
@@ -63,7 +64,7 @@ def update(self):
                 query = {'object_provides': IEvent.__identifier__}
             else:
                 query = {'portal_type': 'Event'}
-            self.events = context.queryCatalog(**query)
+            self.events = api.content.find(context=context, **query)
```

Comment:
collection.queryCatalog takes the collection criteria into account and adds additional query parameters. 
this way the collection criteria get lost and you limit the results to the path of the collection